lib/dotiw/time_hash.rb

```diff
@@ -11,8 +11,17 @@ def initialize(distance, from_time = nil, to_time = nil, options = {})
       self.options    = options
       self.distance   = distance
       self.from_time  = from_time || Time.now
-      self.to_time    = to_time   || (self.from_time + self.distance.seconds)
+      self.to_time    = to_time   || (@to_time_not_given = true && self.from_time + self.distance.seconds)
```

Comment:
Are you sure about that one? ist that single "=" really meant to be a assignment or is that a mistake? 
It seems like an error or at least misleading style and could be fixed.

(I didnt test anything, just still watching the project, and that is why I truly ask for intention)